### ISSUE TITLE
Authorization error while contacting pods

### ISSUE THREAD

**Issue by amnonkhen** [ebi-ait/dcp-ingest-central#970]
I started getting the following errors on prod without changing anything on the cluster.

```
❯ kubectl logs rabbit-0
Error from server (InternalError): Internal error occurred: Authorization error (user=kube-apiserver-kubelet-client, verb=get, resource=nodes, subresource=proxy)
```
```
❯ kubectl port-forward rabbit-0 15672:15672
error: error upgrading connection: unable to upgrade connection: Authorization error (user=kube-apiserver-kubelet-client, verb=create, resource=nodes, subresource=proxy)
```

Created [support case 13346330601](https://support.console.aws.amazon.com/support/home?region=us-east-1#/case/?displayId=13346330601&language=en) with aws.

**Comment by amnonkhen** [comment]
pods need to be migrated to new nodes running new kubelet.

This [stackoverflow](https://stackoverflow.com/questions/75617980/getting-authorization-error-after-upgrading-eks-to-version-1-25-user-kube-apise) might be relevant.

**Comment by amnonkhen** [comment]
solved with the cluster upgrade task #774
